### ISSUE TITLE
character property 'living' is not set false upon death, or processed

### ISSUE THREAD

**Issue by crescojeff** [crescojeff/Fuzziolump#17]

**Comment by crescojeff** [comment]
mostly addressed in 2c592ba3d43f97fcf8bb54e5fbe496e47485cf41 -- I just saw The Human persist as a zombie after being slain by Puck's Attack abl

**Comment by crescojeff** [comment]
```
Attack killed The Human
ran enemy AI for Puck
enemy HP is 305.51266850963924
The Human's HP is -4.2798163737064066
Dead players: 1, dead enemies: 0
findFirstLivingCharacter; character with id player is living
The Human first incarnation id is shadowform
```
snrrrg...

**Comment by crescojeff** [comment]
`<<elseif $combatArena.turnGroup is "enemy">>
  <<link [["Rally!"->Combat]]>>
  	<<run console.log("enemy HP is "+$combatArena.enemyParty[0].stats.hp);>>
	<<run console.log("The Human's HP is "+$characters["player"].stats.hp)>>
  	<<if $combatArena.processRoundBottom()>>
  		<<set $combatArena.turnGroup = "player">>
		/*<<set $combatArena.turnOwner = "player">>*/
		<<set $combatArena.turnOwner = $findFirstLivingCharacter($combatArena.playerParty).id>>`
whoops.  I guess I was thinking that I only needed array count, indices, and id in the find*Living functions, but actually I look at their living property too, which again shows $combatArena.playerParty to be out of sync with $characters

**Comment by crescojeff** [comment]
fixed in 99e82e4c863d89fda989c31656b1491d31712c1d
